src/test/java/org/broadinstitute/consent/http/service/NihServiceTest.java

```diff
@@ -47,43 +46,64 @@ public void setUp() throws Exception {
     }
 
     private void initService() {
-        service = new NihService(researcherService, libraryCardDAO, userDAO);
+        service = new NihService(researcherService, userDAO, nihServiceDAO);
     }
 
-    @Test
+    @Test (expected = NotFoundException.class)
     public void testAuthenticateNih_InvalidUser() {
         initService();
-        try {
-            service.authenticateNih(new NIHUserAccount(), new AuthUser("test@test.com"), 1);
-            assert false;
-        } catch (BadRequestException bre) {
-            assert true;
-        }
+        service.authenticateNih(nihUserAccount, new AuthUser("test@test.com"), 1);
     }
 
     @Test
     public void testAuthenticateNih() {
         List<UserProperty> props = Collections.singletonList(new UserProperty(1, 1, "test", "value"));
-        when(researcherService.updateProperties(any(), any(),any())).thenReturn(props);
+        when(researcherService.describeUserProperties(any())).thenReturn(props);
+        User user = new User();
+        user.setUserId(1);
+        when(userDAO.findUserById(any())).thenReturn(user);
         initService();
         try {
             List<UserProperty> properties = service.authenticateNih(nihUserAccount, authUser, 1);
             assertEquals(1, properties.size());
             assertEquals(Integer.valueOf(1), properties.get(0).getPropertyId());
```

Comment:
should check: `serviceDAO.updateUserNihStatus` is called with user that has updated properties.